packages/grafana-ui/src/components/VizRepeater/VizRepeater.tsx

```diff
@@ -152,7 +152,6 @@ export class VizRepeater<V, D = {}> extends PureComponent<Props<V, D>, State<V>>
 
     const repeaterStyle: React.CSSProperties = {
       display: 'flex',
-      overflow: minVizHeight ? 'hidden auto' : 'hidden',
```

Comment:
Note: this should be fine since I have added the `overflow: hidden` to the `.panel-content` and the only consumer of the `VizRepater` that use the `minVizHeight` was the bar gauge.